### PR TITLE
Issue170b

### PR DESCRIPTION
**Pull request by Ghost---Shadow**

### CODE REVIEW

File: cmd/import.go [hunk 1]
```diff
@@ -130,6 +136,8 @@ func ImportFromPlan(provider terraform_utils.ProviderGenerator, plan *ImportPlan
 		log.Println(provider.GetName() + " save " + serviceName)
 		// Print HCL files for Resources
 		path := Path(options.PathPattern, provider.GetName(), serviceName, options.PathOutput)
+		log.Println("path" + path)
```
Comment (sergeylanzman) [inline cmd/import.go]: delete

File: cmd/import.go [hunk 2]
```diff
@@ -130,6 +136,8 @@ func ImportFromPlan(provider terraform_utils.ProviderGenerator, plan *ImportPlan
...
+		log.Printf("resource count %d", len(resources))
```
Comment (sergeylanzman) [inline cmd/import.go]: delete

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,7 @@
+provider "aws" {
```
Comment (sergeylanzman) [inline main.tf]: delete this file

File: providers/alicloud/alicloud_provider.go [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+		// "language":          cty.StringVal("en"),
+		// "site":              cty.StringVal(""),
+		// "retry_timeout":     cty.NumberIntVal(0),
+		// "retry_count":       cty.NumberIntVal(0),
+	})
+}
+
+func (p AlicloudProvider) GetResourceConnections() map[string]map[string][]string {
+	return map[string]map[string][]string{
+		// "subnet":         {"vpc": []string{"vpc_id", "id"}},
```
Comment (sergeylanzman) [inline providers/alicloud/alicloud_provider.go]: delete or replace with alicloud connections

File: providers/alicloud/alicloud_provider.go [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+}
+
+const alicloudProviderVersion = ">1.56.0"
+
+// GetConfig return map of provider config for Datadog
+func (p *AlicloudProvider) GetConfig() cty.Value {
+	config, _ := LoadConfigFromProfile()
+
+	return cty.ObjectVal(map[string]cty.Value{
+		// "name":               cty.StringVal("default"),
```
Comment (sergeylanzman) [inline providers/alicloud/alicloud_provider.go]: delete all not used lines

File: providers/alicloud/alicloud_provider.go [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+		"profile": p.profile,
+	})
+	s, _ := json.MarshalIndent(p.Service, "", "\t")
+	fmt.Println("InitService" + string(s))
+	return nil
+}
+
+func (p *AlicloudProvider) GetSupportedService() map[string]terraform_utils.ServiceGenerator {
+	return map[string]terraform_utils.ServiceGenerator{
+		// "vpc":            &VpcGenerator{},
```
Comment (sergeylanzman) [inline providers/alicloud/alicloud_provider.go]: delete all unused lines

File: providers/alicloud/alicloud_provider.go [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+	"fmt"
+	"os"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+	"github.com/zclconf/go-cty/cty"
+
+	"github.com/pkg/errors"
+)
+
+type AlicloudProvider struct {
```
Comment (sergeylanzman) [inline providers/alicloud/alicloud_provider.go]: change to `AliCloudProvider`

File: providers/alicloud/alicloud_service.go [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+	conf, err := LoadConfigFromProfile()
+	if err != nil {
+		return nil, err
+	}
+	return conf.Client()
+}
+
+// LoadConfigFromProfile Loads profile from ~/.aliyun/config.json
+func LoadConfigFromProfile() (*connectivity.Config, error) {
+	fmt.Println("Here")
```
Comment (sergeylanzman) [inline providers/alicloud/alicloud_service.go]: delete

File: providers/alicloud/alicloud_service.go [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+	if runtime.GOOS == "windows" {
+		profilePath = fmt.Sprintf("%s/.aliyun/config.json", os.Getenv("USERPROFILE"))
+	} else {
+		profilePath = fmt.Sprintf("%s/.aliyun/config.json", os.Getenv("HOME"))
+	}
+
+	// Make sure the profile exists
+	_, err := os.Stat(profilePath)
+	if os.IsNotExist(err) {
+		fmt.Println("error")
```
Comment (sergeylanzman) [inline providers/alicloud/alicloud_service.go]: delete 2 lines with fmt.Println

File: providers/aws/aws_provider.go [hunk 10]
```diff
@@ -127,6 +129,8 @@ func (p *AWSProvider) InitService(serviceName string) error {
 		"region":  p.region,
 		"profile": p.profile,
 	})
+	s, _ := json.MarshalIndent(p.Service, "", "\t")
```
Comment (sergeylanzman) [inline providers/aws/aws_provider.go]: delete

File: scaratch [hunk 11]
```diff
@@ -0,0 +1,5 @@
+go build -v && ./terraformer import alicloud --resources=oss --regions=ap-southeast-3
```
Comment (sergeylanzman) [inline scaratch]: delete this file

**Comment by Ghost---Shadow** [comment]
Sorry, didnt mean to raise pull request. This is a work in progress.

**Pull request closed without merging**